surefire-its/src/test/java/org/apache/maven/surefire/its/JUnitPlatformEnginesIT.java

```diff
@@ -77,13 +77,14 @@
     {
         ArrayList<Object[]> args = new ArrayList<>();
         args.add( new Object[] {"1.0.3", "5.0.3", "1.0.0", "1.0.0"} );
-        args.add( new Object[] {"1.1.1", "5.1.1", "1.0.0", "1.0.0"} );
-        args.add( new Object[] {"1.2.0", "5.2.0", "1.1.0", "1.0.0"} );
+        //args.add( new Object[] {"1.1.1", "5.1.1", "1.0.0", "1.0.0"} );
+        //args.add( new Object[] {"1.2.0", "5.2.0", "1.1.0", "1.0.0"} );
         args.add( new Object[] {"1.3.2", "5.3.2", "1.1.1", "1.0.0"} );
-        args.add( new Object[] {"1.4.2", "5.4.2", "1.1.1", "1.0.0"} );
-        args.add( new Object[] {"1.5.2", "5.5.2", "1.2.0", "1.1.0"} );
+        //args.add( new Object[] {"1.4.2", "5.4.2", "1.1.1", "1.0.0"} );
+        //args.add( new Object[] {"1.5.2", "5.5.2", "1.2.0", "1.1.0"} );
         args.add( new Object[] {"1.6.2", "5.6.2", "1.2.0", "1.1.0"} );
-        //args.add( new Object[] { "1.6.0-SNAPSHOT", "5.6.0-SNAPSHOT", "1.2.0", "1.1.0" } );
+        //args.add( new Object[] {"1.7.2", "5.7.2", "1.2.0", "1.1.0"} );
+        args.add( new Object[] {"1.8.2", "5.8.2", "1.2.0", "1.1.2"} );
```

Comment:
Dead code should be removed ... or commented why and when we want to restore it.

Reply:
My motivation was to remove some config because the uild takes long and these configs have been tested already long enough in our tests.